config/config.go

```diff
@@ -353,10 +353,10 @@ func getNetworkConfig(
 			ReadHandshakeTimeout: v.GetDuration(NetworkReadHandshakeTimeoutKey),
 		},
 
-		PeerListGossipConfig: network.PeerListGossipConfig{
-			PeerListNumValidatorIPs: v.GetUint32(NetworkPeerListNumValidatorIPsKey),
-			PeerListPullGossipFreq:  v.GetDuration(NetworkPeerListPullGossipFreqKey),
-			PeerListBloomResetFreq:  v.GetDuration(NetworkPeerListBloomResetFreqKey),
+		PeersGossipConfig: network.PeersGossipConfig{
```

Comment:
Applicable to the similar spots in this PR but we should name this `PeerGossipConfig` since `PeersGossipConfig` reads awkwardly.

Reply:
Thank you for reviewing. 
I am not a native English speaker, so I'm not very familiar with it. 
I have made the changes.